src/List/ListItemText.js

```diff
@@ -20,6 +20,11 @@ export const styleSheet = createStyleSheet('MuiListItemText', () => {
         paddingLeft: 56,
       },
     },
+    dense: {
+      '& > *': {
```

Comment:
I was thinking about a simple CSS inheritance. Do we need to target the children elements?
I think that it would be better that the fontSize naturally inherent. Maybe we can't I haven't looked in detail.